src/main/java/lotto/utils/LottoCompare.java

```diff
@@ -0,0 +1,46 @@
+package lotto.utils;
+
+import java.util.Arrays;
+
+public enum LottoCompare {
+    FIRST("6개 일치 (2,000,000,000원)", 6, false, 2000000000),
```

Comment:
이와 같이 숫자의 자리수가 많아질 때는 `_`를 통해 자리수 구분이 가능합니다!
```suggestion
    FIRST("6개 일치 (2,000,000,000원)", 6, false, 2_000_000_000),
```

---

그리고, `label`에서 사용되고 있는 일치하는 값, 상금과 같은 정보들을 이미 해당 객체가 가지고 있는데, 저는 이 경우 역시 코드의 중복으로 생각하는 편이에요.
`label`의 String을 `String.format()`을 사용해 보는 건 어떨까요?